site/en/docs/privacy-sandbox/fedcm/index.md

```diff
@@ -16,13 +16,15 @@ authors:
 This document outlines a new proposal for identity federation: the Federated
 Credential Management API (FedCM).
 
-*  The [FedCM proposal](https://github.com/fedidcg/FedCM) has entered [public
+*  The [FedCM proposal](https://github.com/fedidcg/FedCM) is open for [public
```

Comment:
We should probably highlight the potential backwards incompatible change in `Implementation status`.

Reply:
And link the mailing group as well.

Reply:
I have this fixed on my laptop. Will push the change once I get to it.

Reply:
Thanks for highlighting the potential change. Do you think it's worth adding the Google Group mailing list?

Reply:
Added.